examples/amphtml-ads/dianomi.a4a.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
```

Comment:
thanks for adding this example, looking good!

Reply:
The two changes below will get rid of the storage error in the console.

Reply:
I have implemented the two changes below and am no longer seeing the storage error in the console.